README.rst

```diff
@@ -1,14 +1,17 @@
 nested_lookup
 #############
 
-.. image:: https://img.shields.io/badge/pypi-0.2.01-green.svg
+.. image:: https://img.shields.io/badge/pypi-0.3.0-green.svg
   :target: https://pypi.python.org/pypi/nested-lookup
 
-A small Python library which enables:
+Python library which enables:
 
-#. key lookups on deeply nested documents.
-#. fetching all keys from a nested dictionary.
-#. get the number of occurrences of a key/value from a nested dictionary
+#. (nested_lookup) key lookups on deeply nested documents.
+#. (get_all_keys) fetching all keys from a nested dictionary.
+#. (get_occurrence_of_key/get_occurrence_of_value) get the number of occurrences of a key/value from a nested dictionary
+#. (nested_get) Get a value in a nested document using its key
```

Comment:
We need to make it very clear that `nested_get` is a small wrapper around `nested_lookup` which simply returns the first value or `None`. Additionally, if the deeply nested document has multiple matches for a key with different values, we have no way of controlling the order (dictionaries do not keep track of order). Are we sure we want to expose this and take responsibility?

Reply:
You are correct will remove this method.